package.json

```diff
@@ -3,8 +3,12 @@
   "version": "5.0.0",
   "description": "Isomorphic implementation of WebSocket",
   "main": "node.js",
+  "exports": {
+    "browser": "./browser.js",
+    "import": "./node.mjs",
+    "require": "./node.js"
+  },
   "types": "index.d.ts",
-  "browser": "browser.js",
```

Comment:
Please left the `browser` field to keep backwards compatibility, it's not standard but some old tools could still be using it.

Reply:
I'm ok with this if that is what the package maintainer wants.

If they do, this PR has "allow edits by maintainers" enabled and should be an easy fix.